src/blueprints/tv2/timeline-object-factories/tv2-atem-video-mixer-timeline-object-factory.ts

```diff
@@ -257,4 +256,22 @@ export class Tv2AtemVideoMixerTimelineObjectFactory implements Tv2VideoMixerTime
     }
     return this.createTransitionEffectTimelineObject(meContent)
   }
+
+  public createAuxTimelineObject(sourceInput: number, layer: Tv2VideoMixerLayer): AtemAuxTimelineObject {
+    return {
+      id: `${ATEM_PREFIX}${layer}_input_${sourceInput}_timelineObject`,
```

Comment:
Any reason why this one is suffixed `_timelineObject`, when other timeline objects created in this file does not suffix that to their id's?

Reply:
No specific reason. We haven't been consistent in formatting our ids for Actions and TimelineObjects...

Reply:
Might be worth looking into at some point. Not the end of the world for now, would just be good eventually.